testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/appclient/basic/AbstractSimpleApplicationClientTestCase.java

```diff
@@ -85,4 +87,22 @@ public void testAppClientJBossDescriptor() throws Exception {
             wrapper.quit();
         }
     }
+
+    @AfterClass
+    public static void afterClass() {
+        String appClientDir = System.getProperty("jboss.dist") + File.separator + "appclient";
+        deleteDirectory(new File(appClientDir + File.separator + "tmp"));
+        deleteDirectory(new File(appClientDir + File.separator + "data"));
+    }
```

Comment:
I may be misunderstanding the JIRA, but I don't think we want to do this. It's possible there is useful data in here if tests fail.

Reply:
If what the PR is doing is cleaning up data in build/target/wildfly-xxx, that's not what I'm looking for. There should be nothing written there.

Ideally the test doesn't deal with that directory at all.